pyfr/backends/openmp/base.py

```diff
@@ -41,14 +46,14 @@ def __init__(self, cfg):
         self._providers = [k(self) for k in kprovcls]
 
         # Instantiate optional kernel provider classes
-        for k in [xsmm.OpenMPXSMMKernels, gimmik.OpenMPGiMMiKKernels,
-                  cblas.OpenMPCBLASKernels]:
-            try:
-                self._providers.append(k(self))
-            except (KeyboardInterrupt, SystemExit):
-                raise
-            except:
-                pass
+        try:
+            self._providers.append(xsmm.OpenMPXSMMKernels(self))
+        except (KeyboardInterrupt, SystemExit):
```

Comment:
We can probably be a bit more direct about which exception we catch.  See what's thrown when we can't load/find xsmm.

Reply:
OSError is thrown when we can't find xsmm. I added an exception for this below.

Reply:
Okay, I think that is the only exception we need to catch. Can remove the other block.